.circleci/config.yml

```diff
@@ -16,13 +16,13 @@ jobs:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "yarn.lock" }}
-      - run: yarn run quality
-      - run: yarn run unit-test
+      - run: yarn quality
+      - run: yarn unit-test
       - run:
-          command: yarn run dev
+          command: yarn dev
           background: true
-      - run: yarn run chimp-test
-      - run: yarn run build
+      - run: yarn chimp-test
+      - run: yarn build
```

Comment:
@jellegerbrandy I like to keep the `run` here and in all of our commands. Reason being if `yarn` ever has a built-in command with the same name as one of ours in `package.json`, then the `yarn` built-in will run instead of ours. We had this problem with `yarn clean` a week or two ago; safest to always include `run` for that reason (unless removing `run` adds some benefit I'm not aware of)

Reply:
good point, will change back